TAA/TAA.md

```diff
@@ -0,0 +1,141 @@
+* e. Waiver​. The waiver by either Party of a breach, default, delay or omission of any of the provisions of this Agreement by the other Party will not be construed as a waiver of any subsequent breach of the same or other provisions.
+* f. Entire Agreement​. This Agreement, including all documents incorporated into this Agreement by reference, constitutes the entire agreement of the Parties with respect to the subject matter of this Agreement, and supersedes any and all prior agreements and understandings of the Parties, whether written or oral, with respect to such subject matter. This Agreement supersedes all prior or Transaction Author Agreements between the Sovrin Foundation and the Transaction Author with respect to the subject matter hereof.
+* g. Modification of This Agreement​. Sovrin Foundation reserves the right to modify this Agreement at any time in accordance with this provision, including, but not limited to, changes in applicable law or guidance from any jurisdiction. Sovrin Foundation will post an amended version of this Agreement on its website at least forty-five (45) days prior to the date on which all Transaction Authors must begin operating under the amendment (the "**Amendment Cutover Date**"). If Transaction Author continues to Author Transactions to the Sovrin Ledger Networks after the Amendment Cutover Date, such continued use will constitute acceptance of the amended Agreement.
+* h. Counterparts​. This Agreement may be executed in two or more counterparts, each of which will be deemed an original, but all of which taken together will constitute one and the same instrument
+* i. Survival​. Any terms that by their nature survive termination or expiration of this Agreement shall survive.
+* j. Governmental Entities​. If Transaction Author is a governmental entity and it determines that GDPR does not apply to it and its Processing of Transactions, then:
+    * a. to the extent that the GDPR requirements referenced in this Agreement are equivalent to the requirements under Data Protection Laws in its own jurisdiction, it will comply with any such requirements; and
+    * b. to the extent that GDPR requirements referenced in this Agreement differ from requirements under Data Protection Laws in its own jurisdiction, it will comply with the requirements under its own legislation.
+
+## **Appendix A**
+**Steward Data Processing Agreement**
+[LINK​](https://sovrin.org/wp-content/uploads/Steward-Data-Processing-Agreement-V1.pdf) AS OF THE EFFECTIVE DATE OF THIS TRANSACTION AUTHOR AGREEMENT
+
+**Appendix B**
+**Transaction Endorser Data Processing Agreement**
+[LINK](https://sovrin.org/wp-content/uploads/Transaction-Endorser-Data-Processing-Agreement-V1.pdf)​ AS OF THE EFFECTIVE DATE OF THIS TRANSACTION AUTHOR AGREEMENT
+
+© 2019 by Sovrin Foundation. This work is licensed under the Creative Commons Attribution-ShareAlike 4.0 International License (​http://creativecommons.org/licenses/by-sa/4.0/​).
```

Comment:
Are we restricted to ASCII? maybe the © should be replaced with `(c)` if we are?

Reply:
I thought "copyright" had an ASCII symbol, something like 0169. I think it's better to play it safe, let's go with `(c)`.